d2renderers/d2sketch/sketch.go

```diff
@@ -53,78 +52,104 @@ func InitSketchVM() (*Runner, error) {
+	</pattern>`, luminanceCategory, fill)
 }
 
 func Rect(r *Runner, shape d2target.Shape) (string, error) {
 	js := fmt.Sprintf(`node = rc.rectangle(0, 0, %d, %d, {
-		fill: "%s",
-		stroke: "%s",
+		fill: "#000",
+		stroke: "#000",
```

Comment:
why are these just black? can we just omit them?

Reply:
I think I've done that and it broke the rendering process. Maybe use a constant that would resolve to `#000`? But that would be a useless overkill. I'll probably just leave a comment somewhere in the file explaining it.

Reply:
I can't remember ATM but I'll try to remove it when I get to my laptop and see what happens. Don't have much time today tho:(

Reply:
this is what happens when I remove it, looks like it doesn't affect all shapes but it sure is broken for some tests

## broken
![overlay](https://user-images.githubusercontent.com/13472046/219956367-1388c8e2-8a75-4319-abb5-6e375268036b.svg)

## original
![overlay_working](https://user-images.githubusercontent.com/13472046/219956490-9608b03d-285d-4003-b8bc-a1e100b9ac75.svg)


## broken
![all_shapes](https://user-images.githubusercontent.com/13472046/219956371-d7ae8da6-7437-474f-95d7-ff5181c8c7c6.svg)

## original
![all_shapes_working](https://user-images.githubusercontent.com/13472046/219956497-f2a23c45-0350-418c-9678-af151fae41f5.svg)

Reply:
weird, those shapes are also different colored for some reason though. perhaps it's blending multiple layers or something